### ISSUE TITLE
Sample course: assessment article trickle missing _onChildren

### ISSUE THREAD

**Issue by swashbuck** [adaptlearning/adapt_framework#3627]
### Subject of the enhancement
In the sample course, there is an assessment on the a-300 article. Trickle is enabled here, but the `_onChildren` property is not explicitly set. When not included on an article's `_trickle`, `_onChildren` behaves as if set to `true` and child blocks are locked one by one. Oddly, the [schema](https://github.com/adaptlearning/adapt-contrib-trickle/blob/fc51663925d2a100ca9fbbefc33d6861b5983330/schema/article.schema.json#L39) defaults to false which seems to be counterintuitive.

Due to this, a problem can occur when importing a framework course into the AAT as the `_onChildren` value will be set to false after import. This causes a discrepancy since the course developer expects Trickle to function as it does in the framework. While not a breaking issue by any means, it can require additional troubleshooting time.

TL;DR: I suggest adding `"_onChildren": true` to the a-300 article's `_trickle` settings.
